src/Graph/GraphFormatter.php

```diff
@@ -151,6 +151,11 @@ public function buildGraph($nodes){
 							}
 						}
 
+						// change arrowhead of edges
```

Comment:
By now you can probably guess how I feel about this comment :)

If the code itself is not clear enough, then extract the if statement into a changeArrowhead (or similar) method. Having a trillion low level details in this function is not good anyway.